corelib/crayfish_capi.cpp

```diff
@@ -409,6 +410,27 @@ bool CF_Mesh_calc_derived_dataset(MeshH mesh,
     }
 }
 
+bool CF_Mesh_calc_derived_dataset_with_mask(MeshH mesh,
+                                  const char* expression,
+                                  float startTime, float endTime,
+                                  const char* maskWkt,
+                                  bool addToMesh, const char* output_filename)
+{
+    QString exp = QString::fromAscii(expression);
+    QString outputFile = QString::fromAscii(output_filename);
+
```

Comment:
Can you convert masWkt here to QString similarly to other inputs and use QString everywhere else in the C++ code?